package.json

```diff
@@ -95,27 +95,27 @@
     "babel-preset-react-optimize": "^1.0.1",
```

Comment:
please remove this change as I think nothing was actually changed

Reply:
Done.